Select depends on popover

When installing select via npm, I get the following error:

```
ERROR in ./~/@angular2-mdl-ext/select/select.js
Module not found: Error: Can't resolve '../popover' in '/home/simon/Projekte/stay-and-charge-app/node_modules/@angular2-mdl-ext/select'
```

Installing popover solves the issue.

--- a/src/components/select/README.md
+++ b/src/components/select/README.md
@@ -2,7 +2,7 @@
 
 ### Installing
 
-    npm i --save @angular2-mdl-ext/select
+    npm i --save @angular2-mdl-ext/popover @angular2-mdl-ext/select
 
     import { MdlSelectModule } from '@angular2-mdl-ext/select';
 

--- a/src/components/select/package.json
+++ b/src/components/select/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@angular2-mdl-ext/select",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "description": "Angular 2 Material Design Lite - Select Component",
   "main": "./index.js",
   "typings": "./index.d.ts",
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/mseemann/angular2-mdl-ext#readme",
   "peerDependencies": {
-    "angular2-mdl": "1.7.1"
+    "angular2-mdl": "1.7.1",
+    "@angular2-mdl-ext/popover": "*"
   }
 }

